recipes/libsecp256k1-py-bindings/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "libsecp256k1-py-bindings" %}
+{% set version = "0.4.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/MementoRC/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: facbc0d35df65eaa7cc999edce08ba3cfd84fa4ff690da1271357f8f4c119155
+
+build:
+  number: 0
+  script:  {{ PYTHON }} -m pip install . --use-pep517 -vvv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - cffi >=1.3.0
+    - libsecp256k1
+    - pip
+    - pkgconfig
+    - python
+  run:
+    - asn1crypto
+    - {{ pin_compatible('cffi') }}
+    - python
+
+test:
+  imports:
+    - libsecp256k1_py_bindings
+  source_files:
+    - tests/
+  requires:
+    - pip
+    - pytest
+    - pytest-benchmark
+  commands:
+    - pip check
+    - pytest -v tests
+
+about:
+  home: https://github.com/MementoRC/{{ name }}
```

Comment:
Please don't use variables in the about section urls because then you can't right click to open them directly.

Reply:
Done